### DISCUSSION TITLE
Sparkos Not syncing

### DISCUSSION THREAD

**Discussion by sh3ikh-faisal** [ROM-builders/temporary#8656] [Q&A] [answered]
I am trying to build from almost 3 days but sparkos is not able to sync can you please look into this.

https://cirrus-ci.com/task/5842720445956096?logs=sync#L0

**Comment by Apon77** [comment, marked as answer]
https://t.me/ROM_builders/72098

> I am trying to build from almost 3 days but sparkos is not able to sync can you please look into this.
> 
> https://cirrus-ci.com/task/5842720445956096?logs=sync#L0

https://t.me/ROM_builders/72098
